### PR TITLE
Pdf transparency

### PR DESCRIPTION
**Pull request by bberenz**
for #952

### CODE REVIEW

File: src/qz/printer/action/PrintPDF.java [hunk 1]
```diff
@@ -82,6 +83,10 @@ public void parseData(JSONArray printData, PrintOptions options) throws JSONExce
                     docHeight = dataOpt.optDouble("pageHeight") * convert;
                 }
 
+                if (!dataOpt.isNull("ignoreTransparency") && dataOpt.optBoolean("ignoreTransparency")) {
+                    ignoreTransparency = true;
+                }
```
Comment (tresf) [inline src/qz/printer/action/PrintPDF.java:88]:
```suggestion
                ignoreTransparency = dataOpt.optBoolean("ignoreTransparency", false);
```

**Comment by klabarge** [comment]
I tested this on macOS using the PDF in https://github.com/qzind/tray/issues/278 and this works great. The left is with `ignoreTransparency: true`, the right is with `ignoreTransparency: false`


image

**Comment by tresf** [comment]
@klabarge thanks!  Will you be able to test on Windows as well as do some basic regression testing?

**Comment by klabarge** [comment]
> Will you be able to test on Windows

I actually couldn't reproduce the poor printing quality in Windows, however I was able to reproduce the slow spooling/printing in #278. `ignoreTransparency: true` fixes the slow printing.

> as well as do some basic regression testing?

I tested backwards and forwards compatibility ( new api vs old app, and vice versa) on Windows with success.

**Pull request merged**